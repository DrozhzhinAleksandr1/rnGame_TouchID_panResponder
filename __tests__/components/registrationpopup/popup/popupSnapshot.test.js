import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PopUp from '../../../../app/components/registrationpopup/popup/popup';

it('create snapshot  PopUp', () => {
    const renderer = new ShallowRenderer()
    let snapshot = renderer.render(<PopUp />);
    expect(snapshot).toMatchSnapshot();
});
