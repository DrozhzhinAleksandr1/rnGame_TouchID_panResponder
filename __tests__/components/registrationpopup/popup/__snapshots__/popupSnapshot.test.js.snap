// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot  PopUp 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(23, 37, 51, 0.8)",
      "elevation": 10,
      "height": "100%",
      "justifyContent": "center",
      "paddingLeft": 16,
      "paddingRight": 16,
      "position": "absolute",
      "width": "100%",
      "zIndex": 10,
    }
  }
>
  <TouchableWithoutFeedback
    accessibilityLabel="BublePopupID"
    accessible={false}
    onPress={[Function]}
    testID="BublePopupID"
  >
    <Component
      style={
        Object {
          "backgroundColor": "transparent",
          "height": "100%",
          "position": "absolute",
          "width": 750,
        }
      }
    />
  </TouchableWithoutFeedback>
  <Component
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1D2E3E",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 20,
          "lineHeight": 32,
          "marginTop": 24,
          "textAlign": "center",
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Light",
          "fontSize": 16,
          "lineHeight": 19,
          "marginTop": 5,
          "textAlign": "center",
          "width": 252,
        }
      }
    />
    <Component
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingBottom": 20,
          "paddingLeft": 24,
          "paddingRight": 24,
          "width": "100%",
        }
      }
    >
      <RegistrationBtn />
      <ExitBtn />
    </Component>
  </Component>
</Component>
`;
